neuroml/nml/helper_methods.py

```diff
@@ -1281,14 +1281,7 @@ def get_ordered_segments_in_groups(self,
                         while par_seg_element!=None:
 
                             par_seg = segments[par_seg_element.segments]
-                            d = par_seg.distal
-                            p = par_seg.proximal
-
-                            if not p:
-                                par_seg_parent_seg = segments[par_seg.parent.segments]
-                                p = par_seg_parent_seg.distal
-
-                            par_length = math.sqrt( (d.x-p.x)**2 + (d.y-p.y)**2 + (d.z-p.z)**2 )
+                            par_length = self.get_segment_length(par_seg.id)
```

Comment:
@pgleeson : replaced with `get_segment_length` here too, but wondering now if there was a reason we were explicitly calculating length here instead of using it in the first place?